addons/udes_stock/models/stock_move_line.py

```diff
@@ -544,7 +544,6 @@ def _prepare_info(self):
                 "product_uom_qty": self.product_uom_qty,
                 "qty_done": self.qty_done,
                 "write_date": self.write_date,
-                "product_packaging": self.product_packaging.name
```

Comment:
This appears to undo a change in an earlier commit from this series: is this deliberate?

Reply:
yes, this was owing to the long conversation we all had about where this flag was desired in the backend.